canonical-websites/snapcraft.io
# Improved store page experience

Fixes https://github.com/canonical-websites/snapcraft.io/issues/1217

More precisely, makes it less obvious...

## QA
- Pull the branch
- `./run`
- Throttle your connection somehow (use devtools, probably the easiest)
- Visit http://0.0.0.0:8004/store
- See featured loaded and placeholders until the others finish


![screenshot_2018-12-13 install linux apps using the snap store snapcraft](https://user-images.githubusercontent.com/479384/49957638-58d16180-ff00-11e8-93d8-0b18753b1b6c.png)

There are many other ways to improve this page and I'm not sure if we should wait for the new designs to be signed off first? I'm open to suggestions.
